frontend/test/features/login_page_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:aurabus/features/loginAndSignUp/presentation/login_page.dart';
+import 'package:aurabus/features/loginAndSignUp/widgets/custom_text_field.dart';
+import 'package:aurabus/features/loginAndSignUp/widgets/generic_button.dart';
+import 'package:aurabus/features/loginAndSignUp/widgets/clickable_text.dart';
+
+void main() {
+  Widget wrap(Widget child) {
+    return MaterialApp(home: child);
+  }
+
+  testWidgets('LoginPage shows titles and fields', (WidgetTester tester) async {
+    await tester.pumpWidget(wrap(const LoginPage()));
+
+    // Header texts
+    expect(find.text('AURABUS'), findsOneWidget);
+    expect(find.text('LogIn'), findsOneWidget);
```

Comment:
it's better to use key class
```suggestion
    expect(find.byKey(const Key('email_field')), findsOneWidget);
    expect(find.byKey(const Key('login_button')), findsOneWidget);
```